R/stochastic.R

```diff
@@ -76,3 +141,13 @@ build_where <- function(filters) {
   }
   where_clause
 }
+
+build_years <- function(year_groups) {
+  if (is.null(year_groups)) {
+    return("")
+  }
+  year_clauses <- lapply(year_groups, function(group) {
+    sprintf("SELECT %s AS start_year, %s AS end_year", min(group), max(group))
```

Comment:
is it ok for start_year to equal end_year here?

Reply:
Yeah they can be the same, so if you pass something like `year_groups = list(2015, 2016, ...)` it will return the same as the `fetch_stochastic_data` function. I should double check that the montagu-reports PR formats those nicely when start and end are the same year.